easybuild/easyblocks/s/suitesparse.py

```diff
@@ -94,8 +94,12 @@ def configure_step(self):
         try:
             for line in fileinput.input(fp, inplace=1, backup='.orig'):
```

Comment:
let's move down the definition of `fp` closer to where it's actually being used, i.e. right above the `try`...